Resources/public/js/components/price-list/main.js

```diff
@@ -57,15 +57,19 @@ define([], function() {
         updateData = function(){
             // get all items
             var $bulkPrices = this.sandbox.dom.children(this.$el),
-                data = [], prices;
+                pricesData = [], prices, specialPrice, specialPriceData = [];
 
             this.sandbox.util.foreach($bulkPrices, function($bulkPrice){
                 prices = this.sandbox.dom.data($bulkPrice, 'items');
-                Array.prototype.push.apply(data, prices);
+                specialPrice = $.data($bulkPrice, 'itemsSpecialPrice');
+                Array.prototype.push.apply(pricesData, prices);
```

Comment:
whats this Array.prototype thing? ...

if you have an array, you can simply push a new element to the end of the array by doing ```priceData.push(prices)```